operating/verify-image-signatures.adoc

```diff
@@ -0,0 +1,41 @@
+[id="verify-image-signatures"]
+= Verifying image signatures
+include::modules/common-attributes.adoc[]
+:context: verify-image-signatures
+
+toc::[]
+
+[role="_abstract"]
```

Comment:
Just a suggestion, but maybe you could use a bulleted list for the next three sentences to list these features.